lib/msf/core/db_manager/import/nmap.rb

```diff
@@ -164,9 +164,12 @@ def import_nmap_xml(args={}, &block)
         data[:host]  = hobj || addr
         data[:info]  = extra if not extra.empty?
         data[:task]  = args[:task]
-        if p["name"] != "unknown"
+        if p["name"]
           data[:name] = p["name"]
         end
+        if p['tunnel']
+          data[:name] = "#{p['tunnel']}/#{data['name'] || 'unknown'}"
```

Comment:
It can be. This parser's different, so either will work. Lemme consolidate into the previous check. :)